### PR TITLE
Create Animals Table

### PR DESCRIPTION
**Pull request by Efiamotu-1**
## Features Implemented

>- Created a table for animals data
>- Inserted sample animal data
>- Queried the database for results

Screenshots of the required queries:-

3-select

Select-33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,5 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
+# Description
```
Comment (HAMMAS-SALEEM) [inline README.md]: - [ ] Kindly add the name of your project before description like below to make your readme more descriptive 😊

```
# Vet Clinic
# Description
```

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @Efiamotu-1 :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
✔️ Following Gitflow
:heavy_check_mark: Following some requirements

Good job on implementing some of the requested changes, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -43,6 +32,3 @@ Give a ⭐️ if you like this project!
 - Inspiration
 - etc
 
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
```
Comment (HAMMAS-SALEEM) [inline README.md]: - [ ] Kindly add a license file in your project and a license link in your readme which you have removed to make your readme more descriptive and to make your project licensed 🤔

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]

**Review by omarsalem7 — APPROVED** [review]
Hi @Efiamotu-1 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive README
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**